src/middleware.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019 Google LLC
```

Comment:
nit: Can we be a bit more specific with this file? I'd recommend `middlewarePubSub.ts`, `middlewareHelper.ts`, or something similar. We define middleware elsewhere in the repo and may have unrelated to Pub/Sub middleware in the future.

Reply:
I can update this. My thinking was eventually when we have more middleware, we would create `middleware/index.ts` and then imports for this module would stay the same...